### DISCUSSION TITLE
Handle page permission denied when using custom flask-oidc

### DISCUSSION THREAD

**Discussion by pajri** [apache/superset#18345] [Q&A / Help]
Hi
I tried to implement flask-oidc in superset. The implementation is more or less like this : [https://github.com/apache/superset/issues/13806#issuecomment-812028277](https://github.com/apache/superset/issues/13806#issuecomment-812028277) 

When I removed permission to a page (logmodelview/list) it goes to function `def handle_login():`. Is there a way to redirect permission denied page to custom error page instead ?

Thank you

**Comment by villebro** [comment]
@dpgaspar do you know how to do this?
